changelog/change_read_write_attribute_to_be_aware_of_shadowed_methods.md

```diff
@@ -0,0 +1 @@
+* Make `Rails/ReadWriteAttribute` cop aware of shadowing methods. ([@drenmi][])
```

Comment:
Can you add this PR address to the changelog entry?

Reply:
Done! 🙇